PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -119,8 +105,7 @@ import EventBus from '../eventBus'
     name: 'SinglePatientContactInfo',
 	data () {
 		return { 
-			phonetype:['Home', 'Mobile', 'Work'],
-			radios: 'May we contact you for a follow up vaccination?',
+			phoneTypeOptions:['Home', 'Mobile', 'Work'],
 			emailRules: [
```

Comment:
On my browser (1920x1080 Chrome, but not on Firefox) the 'Please provide a valid e-mail address' message is hidden by the footer, so there is no message displayed to the user when the text field turns red.

Not sure how worthwhile this would be to fix.

Reply:
@TomMakes @dmcook5208 Let's just keep this on the back-burner for now, and if it becomes an issue we can address it then. Maybe make a card for this bug?

Reply:
I think actually I made a mistake here and removed the row that allowed the 'Please provide a valid e-mail address' sign to be hidden....